cmd/spruce/main_test.go

```diff
@@ -162,7 +162,8 @@ func TestMain(t *testing.T) {
 			stdout = fmt.Sprintf(format, args...)
 		}
 		var stderr string
-		printfStdErr = func(format string, args ...interface{}) {
+		//Edit log stderr function
+		PrintfStdErr = func(format string, args ...interface{}) {
```

Comment:
Because the warnings print out through the log library, which, in order to be tested, needs to have a hook that we can insert our string into. The warnings, because they're in the spruce package, can't use that function, lest we dependency cycle.